ci/azure-pipelines.yml

```diff
@@ -134,6 +134,23 @@ stages:
             displayName: Build on Ubuntu without kernel module
             condition: contains(variables['build_container'], 'ubuntu')
 
+          - bash: |
+              set -eEx
+              git clean -xdf
+              ./autogen.sh
+              ./configure --disable-kernel-module
+              make dist-gzip
+              mkdir {BUILD,BUILDROOT,RPMS,SOURCES,SPECS,SRPMS}
+              mv xpmem-*.tar.* ./SOURCES/
+              kernel_ver=$(rpm -qa | grep kernel-devel | cut -d'-' -f3-)
```

Comment:
1. what happens if we have multiple packages? maybe iterate over all of them?
2. WDYT `rpm -qa 'kernel-devel*'  --qf "%{version}-%{release}.%{arch}\n"`

Reply:
fixed both

Reply:
for some reason it does not work on our CI, so falling back to other approach (also rpm -qli seems to fail returning error code when .rpm's are not there)